requestrouter.cpp

```diff
@@ -162,6 +156,10 @@ namespace libtremotesf::impl {
             }
             logDebug(" - Timeout: {}", mConfiguration.timeout);
             logDebug(" - HTTP Basic access authentication: {}", mConfiguration.authentication);
+            if (mConfiguration.authentication) {
+                auto base64Credentials = QString("%1:%2").arg(mConfiguration.username, mConfiguration.password).toUtf8().toBase64();
```

Comment:
RFC 7617 says that Normalization Form C should used for Unicode credentials, you can use QString::normalized before calling toUtf8() for that (https://doc.qt.io/qt-6/qstring.html#normalized)